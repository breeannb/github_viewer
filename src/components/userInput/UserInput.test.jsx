import React from 'react';
import { shallow } from 'enzyme';
import UserInput from './UserInput';

describe('App component', () => {
  it('renders App', () => {
    const wrapper = shallow(<UserInput />);
    expect(wrapper).toMatchSnapshot();
  });
});
