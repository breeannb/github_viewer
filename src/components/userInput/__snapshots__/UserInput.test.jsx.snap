// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<form>
  <input
    type="text"
  />
  <button>
    Submit
  </button>
</form>
`;
